keps/sig-node/24-apparmor-ga/README.md

```diff
@@ -0,0 +1,698 @@
+# AppArmor graduation to General Availability (GA)
```

Comment:
```suggestion
# AppArmor security for Pods
```
or
```suggestion
Add AppArmor support
```
The second version is the title of the KEP according to https://github.com/kubernetes/enhancements/issues/24